plugins/services/src/js/containers/volume-detail/VolumeDetail.js

```diff
@@ -54,6 +54,10 @@ class VolumeDetail extends React.Component {
       />
     );
 
+    const sizeLabel = volume.type === 'External'
```

Comment:
Consider the following example which would introduce a server bug that is rather hard to spot during a simple code review. 

```javascript
var foo = bar = 'baz' 
  ? 'qux' : 'quux'
```

Let's please not use ternaries until we have a solid set of eslint rules to cover the bad parts.